.github/files/build-reminder-comment/check-test-reminder-comment.js

```diff
@@ -14,32 +14,39 @@ const getCheckComments = require( './get-check-comments.js' );
  * @param {string} repo   - Repository name.
  * @param {string} number - PR number.
  * @param {Core}   core   - A reference to the @actions/core package
- * @return {Promise} Promise resolving to an array of project strings needing testing.
+ * @return {Promise} Promise resolving to an object with keys `projects` and `plugins`, each being an array of strings needing testing.
```

Comment:
Maybe it should be `packages` instead of `projects` (since the latter seems to be the generic term for plugins and packages)?

That said, I'm not quite sure I understand why we need the distinction in this case. It seems `jetpack` is being pushed to both, `mu-wpcom-plugin` to `projects`, and `wpcomsh` to `plugins`.

And then the returned value seems to be passed down a few times, but I'm not seeing where the difference matters?

Reply:
Yeah, I may as well rename the variables to make more sense.

Reply:
https://github.com/Automattic/jetpack/pull/41879/commits/a2858379e37469dd13b9df7a71084e4259780cab